### ISSUE TITLE
Waterfox 2020-10 issue with Snap Links 3.1.11?

### ISSUE THREAD

**Issue by troOn3000** [cpriest/SnapLinksPlus#388]
Hello!

I just did a refresh install of Waterfox 2020-10, I used frequently Snap Links until now on the same version of Waterfox, but now I'm unable to make it work and I don't know why.

My version of Snap Links is 3.1.11, I still can drag a selection around links, but nothing happens, no links opening. I tried previous versions of Snap Links provided by Firefox add-ons website, same problem.

But for some reasons, the multiple checkboxes function is still ok.

**Comment by troOn3000** [comment]
Yes I tested Google & Wikipedia & other websites, I got nothing. and yes, the link numbers is displayed when I drag a selection.

**Comment by WagnerGMD** [comment]
Even today the trouble remain with [WaterFox Classic](https://github.com/WaterfoxCo/Waterfox-Classic/releases) (the current version is 2022.08) sic...
Let's hope @cpriest is still alive because you're right @troOn3000, the addon **"Snap Links Plus v3.1.8"** is the latest which works fine.

Apparently [there is a plan](https://github.com/cpriest/SnapLinksPlus/milestones) then I'm wonder if you can give us any news (or a little summary) ? Because perhaps there is a trouble with Firefox Nightly (version 106) sic... 
Unless I have a mistake with the settings (inside "about:config") ? For the moment, I can't open any links, neither select the pictures when a website use a security (such as "recaptcha" or "hchaptcha" etc).

PS : If you need more information, let me (us) now. 
`User Agent = Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:68.0) Gecko/20100101 Firefox/68.0 Waterfox/56.6.2022.08`

**Comment by remi-garcia** [comment]
Nor @cpriest or I have had time to put into SLP3 recently. It happens to work fine enough for my usage thus I don't have a need for putting a lot of effort into it, tbh.
However, if anyone feels like doing a PR I would be more than happy to test and review it.

Currently we try to have it working on multiple OS and last Firefox version, which is really not a piece of cake given the recent changes of behavior... Having Waterfox to work would be a bonus...
